src/main/cli/settings.c

```diff
@@ -440,7 +440,10 @@ static const char * const lookupTableLaunchControlMode[] = {
 #endif
 
 static const char * const lookupTableTpaMode[] = {
-    "PD", "D"
+    "PD", "D",
+#ifdef USE_WING    
```

Comment:
trailing whitespaces

Reply:
Would be nice if github would show the space.

Reply:
i was researching git hooks, but they are personal preferences and not included when cloning the repo.  i did not investigate husky.